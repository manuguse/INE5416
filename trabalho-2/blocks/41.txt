1 1 7 11 11 11 11 11
2 2 2 12 13 14 14 11
2 6 6 6 13 14 14 11
3 3 8 8 14 14 14 18
4 4 8 8 15 15 16 18
5 4 9 10 16 16 16 18
5 4 10 10 10 17 18 18
5 4 10 10 10 17 18 18