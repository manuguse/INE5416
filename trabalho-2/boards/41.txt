- - - 2 1 - 3 7
- 3 2 - - 6 4 -
- - 1 3 - - - -
- - - 2 - 2 - 5
- - - - 2 - 3 -
- - - 5 - 4 - -
- - 6 2 - - - -
- 2 4 - - - - -