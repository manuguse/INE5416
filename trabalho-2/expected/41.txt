1 2 1 2 1 5 3 7
4 3 2 1 2 6 4 6
1 2 1 3 1 5 3 4
2 1 4 2 7 2 1 5
1 5 3 1 2 1 3 4
3 4 1 5 1 4 2 3
2 3 6 2 7 2 7 2
1 2 4 1 3 1 6 1